packages/www/docs/guides/media-server.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Media Server
+description: Livepeer.com Media Server documentation
+metaTitle: Media Server - Livepeer.com
+metaDescription: Livepeer.com Media Server documentation
```

Comment:
@MRTIBBETS Should the meta description instead be "Livepeer Media Server documentation".

We should also replace all meta data instances of "Livepeer.com" with "Livepeer Video Services" but that can be done in a follow up PR.